Emux.GameBoy/Cartridge/BufferedExternalMemory.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Emux.GameBoy.Cartridge {
```

Comment:
Please start braces on new line to conform with the style of the rest of the project.